qemu/tests/cfg/mmu_basic.cfg

```diff
@@ -0,0 +1,19 @@
+- mmu_basic:
+    type = mmu_basic
+    only pseries
+    only Host_RHEL.m8
+    no RHEL.6
+    virt_test_type = qemu
+    kill_vm = yes
+    pre_mmu = Radix
+    mmu = Hash
```

Comment:
`mmu` & `pre_mmu` is not clear keys, however, the mmu type also depends on cpu platform.

Reply:
Here is to express the MMU mode before the change and the MMU mode after the change.
So what do you think of pre_mmu_mode and mmu_mode?

Reply:
yes, good idea
correct

Reply:
Update an option, RHEL.7: mmu_option = no